src/coredump/coredump.c

```diff
@@ -24,6 +24,7 @@
 #include "conf-parser.h"
 #include "copy.h"
 #include "coredump-vacuum.h"
+#include "coredump-ratelimit.h"
```

Comment:
sort here too

Reply:
Addressed the comment. Please check once.